pages/landing/hipaa/pricing/_price-rows.html

```diff
@@ -0,0 +1,127 @@
+<tr>
+  <td style="width: 25%">&nbsp;</td>
+  {% for pricing in page.pricing %}
+  <td
+    style="
+      width: 25%;
+      border-left: 1px solid #194c5f;
+      border-right: 1px solid #194c5f;
+      background-color: #252e3a;
+    "
+    class="text-center"
+  >
+    <hr />
+    {% if pricing.price == "Contact Us" %}
+    <p class="h1 enterprise-h1">Contact Us</p>
+    {% elsif pricing.price == "Free" %}
+    <p class="h1 enterprise-h1">Free</p>
+    {% elsif pricing.disabled_price %}
+    <p class="big-price-disabled"><span>$</span>{{ pricing.disabled_price }}</p>
+    <p class="help-block-disabled" style="margin-top: -9px">/ month</p>
+    {% if pricing.disabled_flat_price %}
+    <p class="medium-price-disabled">
+      <span>+ $</span>{{ pricing.disabled_flat_price }}
+      <span class="help-block-disabled">flat fee</span>
+    </p>
+    {% endif %} {% else %}
+    <p class="big-price"><span>$</span>{{ pricing.price }}</p>
+    <p class="help-block" style="margin-top: -9px">/ month</p>
+    {% if pricing.flat_price %}
+    <p class="medium-price">
+      <span>+ $</span>{{ pricing.flat_price }}
+      <span class="help-block">flat fee</span>
```

Comment:
This class includes `display: block`, which is forcing the span onto the next line. Not sure what that class was intended for, but it looks like the only style rule that actually matters here is `font-size: 16px`.

Reply:
Follow up thought. Would it be better to just advertise "no flat fee" instead? I'm not sure myself — just presenting the option. 
<img width="763" alt="image" src="https://user-images.githubusercontent.com/687448/127566504-2b96be0c-f34a-427e-80e4-ea4c0310c60b.png">

Reply:
Ah - thanks! Fixed in 6b347422